services/block_service.go

```diff
@@ -240,17 +228,13 @@ func (s *BlockAPIService) GetBlock(
 	}
 	// Note: We need a full node to return a complete RPCBlock,
 	// otherwise, only body.Hash is populated. body.Transactions is empty.
-	// TODO: log warn if len(body.Hash) > 1 && len(body.txs) == 0
+	// TODO(xiaying): log warn if len(body.Hash) > 1 && len(body.txs) == 0
```

Comment:
make sure to create an issue for this todo.

Reply:
https://jira.coinbase-corp.com/browse/ROSE-568